feat(blog): Added Fine Grained News November 2024 Community Edition and related files (#909)
openfga/openfga.dev: blog/fine-grained-news-2024-11.md

## **Community Highlights**

**OpenFGA at KubeCon:** In November, [Andres Aguiar](https://www.linkedin.com/in/andresaguiar/) represented OpenFGA at KubeCon/CloudNativeCon. OpenFGA had a kiosk in the Project Pavilion, where Andres delivered a lightning talk and participated in *The Policy Engines Showdown* with other authorization solution providers. [Watch the panel discussion...](https://www.youtube.com/watch?v=AVA32aYObRE)

![Andres Aguiar at OpenFGA's KubeCon booth](../static/img/blog/fgn-2024-11-kubecon1.jpg)
![Andres Aguiar participating in The Policy Engines Showdown](../static/img/blog/fgn-2024-11-kubecon2.jpg)

*Andres Aguiar representing OpenFGA at KubeCon*

**OpenFGA in Italy:** [Andrea Chiarelli](https://www.linkedin.com/in/andreachiarelli/) presented *Authorize in the Cloud with OpenFGA* at Cloud Day 2024 in Milan.

![Andrea Chiarelli presenting OpenFGA in Milan](../static/img/blog/fgn-2024-11-andrea-chiarelli1.png)
![Andrea Chiarelli during his talk at Cloud Day 2024](../static/img/blog/fgn-2024-11-andrea-chiarelli2.png)

*Andrea Chiarelli presenting at Cloud Day 2024*

> **OpenFGA Offsite:** The team that works hard to bring you OpenFGA met in Chicago this November for a fun and productive offsite, diving deep into our vision, customer needs, and the roadmap ahead.

Reviewer: Can we use developer or user instead of customer?

- Overcoming Security Challenges in Protecting Shared Generative AI Environments: Explore solutions for ensuring secure, scalable, and efficient multi-tenancy in generative AI environments. [Read more on Medium...](https://towardsdatascience.com/overcoming-security-challenges-in-protecting-shared-generative-ai-environments-1ffb27da1bde)

- Fine-Grained Authorization for Backstage using OpenFGA: Learn how OpenFGA enables dynamic fine-grained authorization in Backstage through ReBAC models and seamless policy updates. [See the webinar on YouTube...](https://www.youtube.com/watch?v=wWFbLPvwOyQ)

## **New Adopters**

- Are you using OpenFGA in production? Join our growing community of adopters! Add your company to our [ADOPTERS.md](https://github.com/openfga/community/blob/main/ADOPTERS.md#companiesprojects-using-openfga-in-production) file with a quick PR.

- Do you offer OpenFGA implementation services? Get listed in our [Implementation Services](https://github.com/openfga/community/blob/main/ADOPTERS.md#companies-offering-openfga-implementation-services) directory. Note: Listings are community-contributed and not officially endorsed by the OpenFGA project.

## **Announcements**

> **OpenFGA Rankked #5 in CNCF Project Contributions!**

Reviewer 2: ```suggestion
**OpenFGA Ranked #5 in CNCF Project Contributions!**
```

---
title: Fine Grained News - November 2024
description: Fine Grained News
slug: fine-grained-news-2024-11
date: 2024-11-30
authors: hello-caleb
tags: [newsletter]
image: https://openfga.dev/img/og-rich-embed.png
hide_table_of_contents: false
---
# Fine Grained News - November 2024

Welcome to the November edition of Fine Grained News! As we enter the final stretch of 2024, there are exciting developments in the OpenFGA to share.

🌟 **We just hit 3,000 stars on the OpenFGA repo!** 🌟 Because of this great community, we've just this incredible milestone! Thank you so much for all the support you've shown this project. Let's keep the momentum going! If you haven't yet, we'd greatly appreciate you [starring the repo](https://github.com/openfga/openfga) to help push us toward 4,000 stars and grow our amazing community! 

![Celebrating OpenFGA reaching 3,000 GitHub stars](../static/img/blog/fgn-2024-11-stars.png)

## Just Shipped

> **v1.8.1:** This release focuses on performance improvements and enhanced monitoring capabilities, including two new flags for better control over check operations, improved performance for TTU relationships with set operations, and expanded metrics tracking with duration measurements. The release also adds deduplication logic to BatchCheck API and introduces new logging fields for authz calls. [Read more in the Changelog...](https://github.com/openfga/openfga/compare/v1.8.0...v1.8.1)

Reviewer: ```suggestion
**v1.8.1:** This release focuses on performance improvements and enhanced monitoring capabilities, including two new flags for better control over check operations, improved performance for TTU relationships with set operations, and expanded metrics tracking with duration measurements. The release also adds deduplication logic to BatchCheck API and introduces new logging fields for authz calls. [Read more in the Changelog...](https://github.com/openfga/openfga/releases/tag/v1.8.1)
```

Author: What was the edit needed here?

For more about the new OPENFGA_CHECK_ITERATOR_TTL and OPENFGA_CHECK_CACHE_LIMIT flags, run `./openfga run --help`  

**Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, and included other performance improvements. [Read more in the Changelog...](https://github.com/openfga/openfga/compare/v1.7.0...v1.8.0)

## **Coming Up**

> **SDK Updates:** We will be updating the SDKs next to take advantage of the new BatchCheck, starting  with Python and JavaScript. If you want to see an SDK prioritized, let us know!

Reviewer: ```suggestion
**SDK Updates:** We will be updating the SDKs next to take advantage of the new BatchCheck, starting with Python and JavaScript. If you want to see an SDK prioritized, let us know!
```

Author: This line has previously been updated to correct a formatting error, so this suggested change may be referencing a change that has already been made.

> **SDK Updates:** We will be updating the SDKs next to take advantage of the new BatchCheck, starting  with Python and JavaScript. If you want to see an SDK prioritized, let us know!

Reviewer: This line is repeated - should something else be here instead?

Reviewer: Maybe the work on the Changelog filter on the SDKs?

Author: ```suggestion
To clarify, were you thinking the changelog filtering work on the `ReadChanges` API that was done in v1.8.0? If so, that maybe better to work into the paragraph on Batch Check since it was the same release.

How about editing the Batch Check paragraph on L.25 like this: 

**Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, time-based filtering in the `ReadChanges` API, and other performance improvements. Read more in the Changelog...
```suggestion
```

Author: I went ahead and added time filtering to the additional update included in v1.8.0 outside of Batch Check.

- **Fine-Grained Authorization for Backstage using OpenFGA:** Learn how OpenFGA enables dynamic fine-grained authorization in Backstage through ReBAC models and seamless policy updates. [See the webinar on YouTube...](https://www.youtube.com/watch?v=wWFbLPvwOyQ)

## **New Adopters**

- Are you using OpenFGA in production? Join our growing community of adopters! Add your company to our [ADOPTERS.md](https://github.com/openfga/community/blob/main/ADOPTERS.md#companiesprojects-using-openfga-in-production) file with a quick PR.

- Do you offer OpenFGA implementation services? Get listed in our [Implementation Services](https://github.com/openfga/community/blob/main/ADOPTERS.md#companies-offering-openfga-implementation-services) directory. Note: Listings are community-contributed and not officially endorsed by the OpenFGA project.

## **Announcements**

> ### **OpenFGA Rankked #5 in CNCF Project Contributions!**

Reviewer: ```suggestion
### **OpenFGA Ranked #5 in CNCF Project Contributions!**
```

Author: This line has previously been updated to correct a formatting error, so this suggested change may be referencing a change that has already been made.

Thanks to our amazing community, OpenFGA soared to become the 5th most active CNCF project during Hacktoberfest in October! Your contributions made this possible, and hope to continue the engagement!

Ready to join our community of contributors? We have opportunities for every skill level:

> - Start with our [Good First Issues](https://github.com/openfga/openfga/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) for beginner-friendly tasks.

Reviewer: Maybe this is a better link?

https://github.com/search?q=org%3Aopenfga+is%3Aopen+label%3A%22good+first+issue%22+&type=issues&s=created&o=desc

as it is not just constrained to the server

Author: Suggestion accepted and link updated.

- Take on more complex challenges in our [Issue queue](https://github.com/openfga/openfga/issues).
- Follow our [Contribution Guide](https://github.com/openfga/.github/blob/main/CONTRIBUTING.md) to get started.

![CNCF projects ranked by commits during Hacktoberfest](../static/img/blog/fgn-2024-11-open-fga-ranks-5th.jpeg)

*CNCF Projects Ranked by Commits during Hacktoberfest*

### **Follow OpenFGA on LinkedIn**

> Connect with a growing community of fine-grained authorization enthusiasts and expand your professional network by following our new <!-- markdown-link-check-disable -->

Reviewer: Unrelated to this post, can we add the LinkedIn page to the community page in docs? https://openfga.dev/docs/community

Author: Added LinkedIn page to Community page. PR: https://github.com/openfga/openfga.dev/pull/912

---
title: Fine Grained News - November 2024
description: Fine Grained News
slug: fine-grained-news-2024-11
date: 2024-11-30
authors: hello-caleb
tags: [newsletter]
image: https://openfga.dev/img/og-rich-embed.png
hide_table_of_contents: false
---
# Fine Grained News - November 2024

Welcome to the November edition of Fine Grained News! As we enter the final stretch of 2024, there are exciting developments in the OpenFGA to share.

🌟 We just hit 3,000 stars on the OpenFGA repo! A huge thank you to our incredible community for your support in building OpenFGA. Let's keep the momentum going! If you haven't yet, we'd greatly appreciate you [starring the repo](https://github.com/openfga/openfga) and pushing us toward 4,000 and growing our community! 🌟
  <p align="center">
  <img src="../static/img/blog/fgn-2024-11-stars.png" alt="OpenFGA 3000 stars" />
</p>

## Just Shipped

> **Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, and included other performance improvements. [Read more in the Changelog...](https://github.com/openfga/openfga/compare/v1.7.0...v1.8.0)

Reviewer: ```suggestion
**Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, and included other performance improvements. [Read more in the Changelog...](https://github.com/openfga/openfga/releases/tag/v1.8.0)
```

Link the actual changelog entry rather than the commits - it's more readable for folks, and they can reach the commits from there

> **Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, and included other performance improvements. [Read more in the Changelog...](https://github.com/openfga/openfga/compare/v1.7.0...v1.8.0)

Reviewer: And let's keep this as 1.8.0, we're linking to 1.8.1 above and 1.8.1 doesn't have the batch check details

> **Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, and included other performance improvements. [Read more in the Changelog...](https://github.com/openfga/openfga/compare/v1.7.0...v1.8.0)

Author: I've listed v1.8.1 (because it's the latest release) and v1.8.0 (because of Batch Check). Let me know if we want to reduce the details or arrange it in some other way.

## Coming Up

**SDK Updates:** We will be updating the SDKs next to take advantage of the new endpoint, starting  with Python and JavaScript. If you want to see an SDK prioritized, let us know!

Check out our roadmap to see what we're working on. Feature requests and ideas can be shared in [GitHub Discussions](https://github.com/orgs/openfga/discussions).

## Community Highlights

* **OpenFGA at KubeCon:** In November, [Andres Aguiar](https://www.linkedin.com/in/andresaguiar/) represented OpenFGA at KubeCon/CloudNativeCon. OpenFGA had a kiosk in the Project Pavilion. Andres delivered a lightning talk on OpenFGA and participated in *The Policy Engines Showdown*, with other authorization solution providers. [See the panel discussion video...](https://www.youtube.com/watch?v=AVA32aYObRE)
<p align="center">
  <img src="../static/img/blog/fgn-2024-11-kubecon1.jpg" alt="Andres Aguiar at OpenFGA's KubeCon booth" />
</p>
 <p align="center">
  <img src="../static/img/blog/fgn-2024-11-kubecon2.jpg" alt="Andres Aguiar at OpenFGA's KubeCon booth" />
</p>
<p align="center">
  <em>Andres Aguiar Representing OpenFGA at KubeCon in Salt Lake City</em>
</p>

> 

Author: Formatting that caused errors previously has been removed, and a preview has been deployed

* **OpenFGA in Italy:** [Andrea Chiarelli](https://www.linkedin.com/in/andreachiarelli/) presented *Authorize in the Cloud with OpenFGA* at November's Cloud Day 2024 in Milan. 
<p align="center">
  <img src="../static/img/blog/fgn-2024-11-andrea-chiarelli1.png" alt="Andrea Chiarelli at Cloud Day 2024" />
</p>
<p align="center">
  <img src="../static/img/blog/fgn-2024-11-andrea-chiarelli2.png" alt="Andrea Chiarelli at Cloud Day 2024" />
  <em>Andrea Chiarelli at Cloud Day 2024</em>
</p>

* **API World:** Our [Tyler Nix](https://www.linkedin.com/in/tylernix/) presented *Leveraging Modern Authorization Models to Scale Your Applications* at API World this month. Which was introduction to modern authorization models and key principles to address the fine-grained authorization challenges.
<p align="center">
  <img src="../static/img/blog/fgn-2024-11-tyler-nix.jpeg" alt="Tyler Nix" />
</p>
<p align="center">
  <em>Tyler Nix</em>
</p>

> * **OpenFGA Offsite:**  [Placeholder text. Add details here]<!--Placeholder. Add team photo(s)and community members photos-->

Author: - Updated the photo only to include the team photo. 
- Added more information to describe the offsite.
